packages/react-components/react-tree/library/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -145,10 +148,57 @@ export function useTreeItem_unstable(props: TreeItemProps, ref: React.Ref<HTMLDi
     });
   });
 
+  const { findFirstFocusable } = useFocusFinders();
+
+  const handleTreeGridActionsKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (!event.currentTarget.contains(event.target as Node) || !treeItemRef.current) {
+      return;
+    }
+    switch (event.key) {
+      case treeDataTypes.ArrowLeft: {
+        // TODO: this should be included in requestTreeResponse navigation signature
+        treeItemRef.current?.focus();
+        return;
+      }
+      case treeDataTypes.ArrowUp: {
+        requestTreeResponse({
+          requestType: 'navigate',
+          event,
+          value,
+          itemType,
+          parentValue: undefined,
+          type: event.key,
+          target: treeItemRef.current,
+        });
+        return;
+      }
+      case treeDataTypes.ArrowDown: {
+        if (!isHTMLElement(event.target) || event.target.hasAttribute('aria-haspopup')) {
+          return;
+        }
```

Comment:
This isn't necessary in the TreeGrid - why is it necessary here?

Reply:
`TreeGrid` is using tabster, this is being resolved by tabster automatically. In this case here with `Tree` we have to do it ourselves as `Tree` component doesn't support tabster at the moment